docs/en/observability/apm/open-telemetry.asciidoc

```diff
@@ -1,51 +1,129 @@
+refer to https://github.com/elastic/opentelemetry[Use OpenTelemetry with Elastic].
+====
 
 Elastic integrates with OpenTelemetry, allowing you to reuse your existing instrumentation
 to easily send observability data to the {stack}.
 There are several ways to integrate OpenTelemetry with the {stack}:
 
-**OpenTelemetry API/SDK with Elastic APM agents**
+* <<apm-otel-elastic-distros,Elastic Distributions for OpenTelemetry>>
+* <<apm-otel-api-sdk,OpenTelemetry API/SDK>>
+* <<apm-otel-collector,OpenTelemetry Collector>>
+* <<apm-otel-api-sdk-elastic-agent,OpenTelemetry API/SDK + Elastic APM agent>>
+* <<apm-otel-lambda,Lambda collector exporter>>
+
+[float]
+[[apm-otel-elastic-distros]]
+== Elastic Distributions for OpenTelemetry
+
+Elastic offers several distributions for OpenTelemetry.
+A _distribution_ is a customized version of an upstream OpenTelemetry repository.
+Each Elastic Distribution for OpenTelemetry is an extension of an https://opentelemetry.io/docs/languages/[OpenTelemetry Language API or SDK].
+
+image::images/apm-otel-distro.png[]
+
+With an Elastic distro you have access to all the features of the OpenTelemetry API or SDK that it extends, plus:
+
+* You may get access to API/SDK improvements and bug fixes contributed by the Elastic team _before_
+the changes are available upstream in the OpenTelemetry repositories.
+* The distro preconfigures the collection of tracing and metrics signals, applying some opinionated defaults,
+such as which sources are collected by default.
 
-To unlock the full power of the {stack}, use the OpenTelemetry API/SDKs with Elastic APM agents,
-currently supported by the Java, Python, .NET, and Node.js agents.
-These Elastic APM agents translate OpenTelemetry API calls to Elastic APM API calls.
-This allows you to reuse your existing instrumentation to create Elastic APM transactions and spans--
-avoiding vendor lock-in and having to redo manual instrumentation.
+// TODO: Update links when distro docs are markdown-ified
+Get started with an Elastic Distribution for OpenTelemetry:
 
-<<apm-open-telemetry-with-elastic,Get started>>.
+* https://github.com/elastic/elastic-otel-java[Java]
+* https://github.com/elastic/elastic-otel-dotnet[.NET]
+* https://github.com/elastic/elastic-otel-node[Node.js]
+* https://github.com/elastic/elastic-otel-python[Python]
 
-**OpenTelemetry agent**
+[NOTE]
+====
+For more details about OpenTelemetry distributions in general,
+visit the https://opentelemetry.io/docs/concepts/distributions[OpenTelemetry documentation].
+====
+
+[float]
+[[apm-otel-api-sdk]]
+== OpenTelemetry API/SDK
 
 The {stack} natively supports the OpenTelemetry protocol (OTLP).
 This means trace data and metrics collected from your applications and infrastructure by an
 OpenTelemetry agent can be sent directly to the {stack}.
 
-<<apm-instrument-apps-otel,Get started>>.
+// Do we want to say anything about why you would/wouldn't choose this
+// method to send data to Elastic?
```

Comment:
This page could be used as an opportunity to help users decide which method to use since they are all listed side-by-side &mdash; a perfect place to compare/contrast their options. Do we want to say anything about the situations in which a user would/wouldn't choose this method to send data to Elastic?

Reply:
I was going to mention this as well. The OTel API/SDK + Elastic APM agent section gives a little more on why you might choose that. This page seems like and entry point for users and providing that info here could be valuable since they might not know which solution will work for them.